modules/monitoring-knative-autoscaler-pods.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="monitoring-knative-autoscaler-pods_{context}"]
+== Monitoring Knative Serving autoscaler pods using the {product-title} monitoring stack
+
+This section provides example  instructions for the visualization of Knative Serving pod autoscaling metrics by using the {product-title} monitoring tools.
+
+.Prerequisites
+
+* You must have the {product-title} monitoring stack installed. See the documentation about https://docs.openshift.com/container-platform/4.1/monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack[Configuring the monitoring stack].
+
+.Procedure
+
+. Navigate to the {product-title} Web console and authenticate.
+. Navigate to "Monitoring" → "Metrics".
+. Enter the "Expression" and select "Run queries". To monitor Knative Serving autoscaler pods, use the example expression.
```

Comment:
Same note here about using `*` around web console elements:

> Enter the *Expression* and select *Run queries*...

Since we end the sentence with:

> ...use the sample expression.

I would change the final punctuation to a semicolon.